terraform/fullnode/gcp/variables.tf

```diff
@@ -148,3 +148,28 @@ variable "enable_kube_state_metrics" {
   description = "Enable kube-state-metrics within monitoring helm chart"
   default     = false
 }
+
+variable "gke_enable_private_nodes" {
+  description = "Enable private nodes for GKE cluster"
+  default     = true
+}
+
+variable "gke_enable_autoscaling" {
```

Comment:
why not default enable?

Reply:
actually, good point

Reply:
I think there's a fear of the autoscaler potentially running up the cost, but that depends on operators knowing to change `gke_autoscaling_max_cpu` and `gke_autoscaling_max_memory`